nekoyume/Assets/_Scripts/UI/Module/EventNotice/NoticeItem.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Nekoyume.UI.Model;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI.Module
+{
+    public class NoticeItem : MonoBehaviour
```

Comment:
It's component for the marked portion.
![image](https://user-images.githubusercontent.com/48484989/206069154-41dc9b7b-5b14-4b27-b3b9-2bd52fb55f56.png)